pkg/controller/volume/persistentvolume/metrics/metrics.go

```diff
@@ -220,15 +261,28 @@ func (collector *pvAndPVCCountCollector) pvCollect(ch chan<- metrics.Metric) {
 func (collector *pvAndPVCCountCollector) pvcCollect(ch chan<- metrics.Metric) {
 	boundNumberByNamespace := make(map[string]int)
 	unboundNumberByNamespace := make(map[string]int)
+	boundNumberByVACAndNamespace := make(map[pvcLabels]int)
+	unboundNumberByVACAndNamespace := make(map[pvcLabels]int)
+
+	// TODO Remove isVolumeAttributesClassEnabled if VolumeAttributesClass feature reaches GA
+	// TODO Q: Is there a correct way to check?
+	isVolumeAttributesClassEnabled := features.FeatureGates().Enabled("VolumeAttributesClass")
```

Comment:
Is there a better way to check this?

Reply:
I think - `utilfeature "k8s.io/apiserver/pkg/util/feature"` is better way.